src/main/flight/mixer.c

```diff
@@ -743,27 +744,118 @@ STATIC_UNIT_TESTED void servoMixer(void)
 #endif
 #endif
 
+void mixConstrainMotorForFailsafeCondition(uint8_t motorIndex)
```

Comment:
yeah, you know what, i did that the first time and then changed my mind.  :smile:  you're right.